connector_ebisumart/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Quartile Limited
+{
+    "name": "Connector Ebisumart",
+    "version": "12.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Connector",
+    "license": "AGPL-3",
+    "depends": ["purchase", "connector_ecommerce", "sale_order_journal"],
+    "data": [
+        "security/ir.model.access.csv",
+        "data/scheduler.xml",
+        "views/account_journal_views.xml",
+        "views/ebisumart_backend_views.xml",
+        # "views/product_category_views.xml",
+        "views/product_product_views.xml",
+        "views/connector_ebisumart_menu_views.xml",
+        "views/res_partner_views.xml",
+    ],
+    "installable": True,
+}
```

Comment:
```suggestion
}

```
Please run pre-commit

Reply:
Can't run pre-commit. There is no dotfiles in this repo at the moment.